src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,30 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => {
+  const {
+    name = 'comment',
+    email,
+    body = '',
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
+        {' by '}
+
+        <a
+          className="CommentInfo__email"
+          href={email ? `mailto:${email}` : '#'}
```

Comment:
If no email is provided, the anchor tag will lead to a non-existing page (#). It's not a very good user experience to click on a link that does nothing. It would be better to render the email as plain text when it is not available.